publication-rest/src/test/java/no/unit/nva/publication/create/CreatePublicationHandlerTest.java

```diff
@@ -553,6 +555,20 @@ void shouldValidateOkIfInstanceTypeIsNotSet() throws IOException {
         assertThat(response.getStatusCode(), is(equalTo(HttpURLConnection.HTTP_CREATED)));
     }
 
+    @Test
+    void shouldReturnBadRequestWhenAttemptingToPersistPublicationWithMultipleCristinIdentifiers() throws Exception {
+        var request = createEmptyPublicationRequest();
+        request.setAdditionalIdentifiers(Set.of(new AdditionalIdentifier("Cristin", randomString()),
+                                                new AdditionalIdentifier("Cristin", randomString())));
```

Comment:
I think the real issue here is that AdditionalIdentifiers, or at least a subset of these are in fact their own entries in DDB. This allows us to look them up and relate them to the publication, e.g.

```
  | publicationid | publicationid | publicationStuff |
  | publicationid | handle        |  |
  | publicationid | handle        |  |
  | publicationid | cristinId     |  |
  | publicationid | doi           |  |
```